src/wp-includes/update.php

```diff
@@ -121,6 +121,7 @@ function wp_version_check( $extra_stats = array(), $force_check = false ) {
 				array(
 					'webp' => isset( $gd_info['WebP Support'] ),
 					'avif' => isset( $gd_info['AVIF Support'] ),
+					'heic' => isset( $gd_info['HEIC Support'] ),
```

Comment:
Nit: The inline doc needs to update at line no. 118 and 131.

`// Add data for GD WebP and AVIF support.`
`// Add data for Imagick WebP and AVIF support.`

Reply:
Good point, thanks for the feedback @mukeshpanchal27 - I will get that updated!